lib/utils/__tests__/isStandardSyntaxCombinator.test.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+const isStandardSyntaxCombinator = require("../isStandardSyntaxCombinator");
+const postcss = require("postcss");
+const selectorParser = require("postcss-selector-parser");
+
+describe("isStandardSyntaxCombinator", () => {
+  it("tag", () => {
+    return rules("a {}", func => {
+      expect(isStandardSyntaxCombinator(func)).toBeTruthy();
+    });
+  });
+  it("descendant", () => {
+    return rules("a b {}", func => {
+      expect(isStandardSyntaxCombinator(func)).toBeTruthy();
+    });
+  });
+  it("descendant tab", () => {
+    return rules("a\tb {}", func => {
+      expect(isStandardSyntaxCombinator(func)).toBeTruthy();
+    });
+  });
+  it("descendant newline", () => {
+    return rules("a\nb {}", func => {
+      expect(isStandardSyntaxCombinator(func)).toBeTruthy();
+    });
+  });
+  it("descendant (double child)", () => {
+    return rules("a >> b {}", func => {
```

Comment:
I can't find it in a [specification](https://www.w3.org/TR/css3-selectors/#combinators). Doesn't look like a standard one.

Reply:
It's a [level 4 one](https://drafts.csswg.org/selectors-4/#descendant-combinators). I'll take it out though